src/ui/settings/components/InstallPluginAlert.tsx

```diff
@@ -0,0 +1,64 @@
+import { clipboard } from "@metro/common";
+import { installPlugin } from "@lib/plugins";
+import { findByProps } from "@metro/filters";
+import { Forms } from "@ui/components";
+
+const { FormInput } = Forms;
+const Alert = findByProps("alertDarkStyles", "alertLightStyles").default;
+
```

Comment:
I am being very picky but I don't like the empty line here - your choice, though.

Reply:
I wanted to keep the components separate from the other stuff